src/b3d/chisight/gen3d/inference_moves.py

```diff
@@ -154,38 +160,43 @@ def propose_a_points_attributes(
     return _propose_a_points_attributes(
         key,
         observed_rgbd_for_point=observed_rgbd_for_point,
-        latent_depth=new_state["pose"].apply(hyperparams["vertices"][vertex_index])[2],
+        latent_rgbd_for_point=jnp.array(
```

Comment:
@nishadgothoskar haha isn't this going to break inference?

Reply:
What I mean is -- don't we need to set the `rgb` correctly here?

Reply:
i think the rgb is ignored